src/main.rs

```diff
@@ -57,7 +57,7 @@ async fn playground_ui() -> impl actix_web::Responder {
                   <body>
 
                     <elements-api
-                      apiDescriptionUrl="{base_path}/spec/v3.json"
+                      apiDescriptionUrl="/eapi/spec/v1/v3.json"
```

Comment:
We should get version name from from env var

Reply:
```
fn get_api_version() -> String {
    std::env::var("API_VERSION").unwrap_or_else(|_| "v1".to_string())
}
```
```
let api_version = get_api_version();
```

```suggestion
                      apiDescriptionUrl="/eapi/spec/{api_version}/v3.json"
```

Reply:
Added a new env var `spec_base_path `